auth-service/src/main/java/hcmus/alumni/authservice/controller/AuthController.java

```diff
@@ -126,4 +142,36 @@ public ResponseEntity<String> verifyAuthorizeCode(@RequestParam String email, @R
 					.body("Send Authorize code failed: " + e.getMessage());
 		}
 	}
+	
+	@PostMapping("/reset-password")
+	public ResponseEntity<String> resetPassword(@RequestParam String email, @RequestParam String oldPassword, @RequestParam String newPassword) {
```

Comment:
```suggestion
	public ResponseEntity<String> resetPassword(@RequestBody ResetPasswordRequestDto req) {
```
Nên dùng `@RequestBody` thay vì `@RequestParam` cho những api ko xử lý file. Dùng request body sẽ ko bị giới hạn bởi url max length, hỗ trợ các cấu trúc phức tạp
B tạo folder `dto` chứa file ResetPasswordRequestDto. File đó sẽ như này
```
@AllArgsConstructor
@NoArgsConstructor
@Data
public class ResetPasswordRequestDto implements Serializable {
    private static final long serialVersionUID = 1L;

    String email;
    String oldPassword;
    String newPassword;
}
```
Spring sẽ tự động map body của request thành class tương ứng